portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -314,6 +314,37 @@ async function loadSearch() {
   insertSearch(otherElements);
 }
 
+/**
+ * Gets links from the navigation bar
+ * Removes link for current page we are on
+ * @return {array}: the links from other pages.
+ */
+function getOtherLinks() {
+  const pagePath = window.location.pathname;
+  const pathSize = pagePath.length;
+
+  let linkElements =
+      document.getElementById('navBar').getElementsByTagName('a');
+  linkElements = Array.from(linkElements);
+  let numLinks = linkElements.length;
+
+  for (let i = 0; i < numLinks; i++) {
```

Comment:
I'm a little concerned with how complex this function is. Aren't we just trying to filter out the current page?

At a high level I imagine the code to look like (forgive my use of Python):
```python
other_links = []
for link in link_elements:
  if same_page(curr_page, link):
    other_links.append(format_link(link))
```

Where same_page() and format_link() are the unique parts.

Reply:
Yeah, I can definitely simplify this function.